lib/msal-common/test/uCache/AccessTokenEntity.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from "chai";
+import { AccessTokenEntity } from "../../src/uCache/entities/AccessTokenEntity";
+import { ATValues } from "./cacheConstants";
+
+describe("AccessTokenEntity.ts Unit Tests", () => {
+
+    it("Verify an AccessTokenEntity entity", () => {
+        const at = new AccessTokenEntity();
+        expect(at instanceof AccessTokenEntity);
```

Comment:
Is there a more meaningful test you can do here?

Reply:
not really. All this UT is to create a specific cacheEntity type.